docs/user/running-cincinnati.md

```diff
@@ -0,0 +1,71 @@
+# Running Cincinnati
+
+## Prerequisite
+
+APP-SRE publishes the Cincinnati container images at [Quay registry](https://quay.io/repository/app-sre/cincinnati)
+
+We need an image tag from the [Quay registry](https://quay.io/repository/app-sre/cincinnati) to use with the deployment config.
+
+The Cincinnati deploymentconfig present in the git repository at [dist/openshift/](https://github.com/openshift/cincinnati/blob/master/dist/openshift/cincinnati.yaml)
+
+### Steps
+
+* Create a new namespace for Cincinnati
+
+```shell
+oc new-project cincinnati
+```
+
+* Create Cincinnati deployment
+
+```shell
+export IMAGE_TAG=<TAG>
+oc new-app -f cincinnati.yaml \
+  -p IMAGE_TAG=${IMAGE_TAG} \
+```
+
+* Wait for teh deployment config to rollout
+
+```shell
+oc wait --for=condition=available --timeout=5m deploymentconfig/cincinnati
+
+```
+
+* Create an route to the Cincinnati policy engine
+
+Example of a yaml for an ingress route to the policy engine
```

Comment:
Is there a reason the route is not part of the `cincinnati.yaml` template?

Reply:
> Is there a reason the route is not part of the `cincinnati.yaml` template?

+1

Reply:
I do not think we should add this as part of deployment config as I am inclined to move to a deployment yaml so that we can deploy it on K8s as well. As deployment is a k8s native concept we should do that.  It will also make life easier for us (developers) as we can easily run K8s locally and test Cincinnati.